zookeeper-server/src/test/java/org/apache/zookeeper/server/DigestCalculatorTestUtil.java

```diff
@@ -0,0 +1,36 @@
+package org.apache.zookeeper.server;
```

Comment:
Please add license header.

Reply:
👍

Reply:
the file has been removed in the latest version